doc/source/user_guide/accessing_results.rst

```diff
@@ -27,22 +25,22 @@ Here is how you get the ``displacement`` result:
 	See the following list for the results that can be
 	extracted.
 
-You can use *keyword arguments* to further specify other options,
+You can use *keyword arguments* to specify additional options,
 including the components, scope, and time. For detailed examples,
 see :ref:`ref_result_keywords`.
 
 PyDPF-Post supports two types of result files:
 
 * Structural (RST)
-* Thermal/electric (RTH) (with the legacy *load_solution()* method only)
+* Thermal/electric (RTH) (with the legacy ``load_solution()`` method only)
```

Comment:
We also support some LSDYNA result files and some Fluent and CFX result files as noted in the getting started

Reply:
@PProfizi Added a sentence with a link to the appropriate topic.